crates/node/src/main.rs

```diff
@@ -201,7 +197,7 @@ async fn main() -> anyhow::Result<()> {
 
     app.start_and_drive_to_end().await?;
 
-    tracing::info!("Shutting down analytics");
+    tracing::info!("🔌 Shutting down analytics...");
```

Comment:
can you remove this log entirely, it's actually not useful tbh

Reply:
This has been fixed.